ERROR: manifest for elasticsearch:latest not found

Hello,

I tried to use this package but i can't seem to make it work for the god of it.
I tried make run, make dev but always this same error.

> docker build -t dsys/match:latest .
Sending build context to Docker daemon  34.43MB
Step 1/7 : FROM python:3.6
 ---> 8256ec07b2ad
Step 2/7 : MAINTAINER Alex Kern <alex@distributedsystems.com>
 ---> Using cache
 ---> 5d02fe4f82ec
Step 3/7 : RUN apt-get update &&     apt-get install -y libopenblas-dev gfortran &&     pip install numpy==1.12.1 &&     pip install scipy==0.19.0 &&     pip install gunicorn==19.7.1 &&     pip install flask==0.12.2 &&     pip install image-match==1.1.2 &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> fe075013d492
Step 4/7 : COPY server.py wait-for-it.sh /
 ---> Using cache
 ---> f050eb22a795
Step 5/7 : EXPOSE 80
 ---> Using cache
 ---> 2ad8be284a7b
Step 6/7 : ENV PORT=80     WORKER_COUNT=4     ELASTICSEARCH_URL=elasticsearch:9200     ELASTICSEARCH_INDEX=images     ELASTICSEARCH_DOC_TYPE=images     ALL_ORIENTATIONS=true
 ---> Using cache
 ---> 2d2c2d902c1e
Step 7/7 : CMD gunicorn     -t 60     --access-logfile -     --access-logformat '%(h)s %(l)s %(u)s %(t)s "%(r)s" %(s)s %(b)s "%(f)s" "%(a)s" - %(D)s'     -b 0.0.0.0:${PORT}     -w ${WORKER_COUNT}     server:app
 ---> Using cache
 ---> 72260f76942c
Successfully built 72260f76942c
Successfully tagged dsys/match:latest
docker-compose up
Pulling elasticsearch (elasticsearch:latest)...
**ERROR: manifest for elasticsearch:latest not found**
Makefile:28: recipe for target 'dev' failed

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -8,6 +8,6 @@ services:
     links:
     - elasticsearch
   elasticsearch:
-    image: elasticsearch
+    image: elasticsearch:6.4.2
     ports:
     - 9200:9200

